### DISCUSSION TITLE
[Inspect Blocked & New-Point Blank Don't work] Securly

### DISCUSSION THREAD

**Discussion by alcradite** [3kh0/ext-remover#677] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Well, my school decided to block the ability to use inspect element and when I try to access the point blank method securely link (even switching to the other extension id that was suggested in the discussion) it will just say "This page has been blocked by Chrome ERR_BLOCKED_BY_CLIENT" even when I click reload it does the same but it goes to invalid (chrome-extension://invalid/) if I click reload, was wondering a fix or a different method?

**Comment by Aka-but-nice** [comment, marked as answer]
https://tinyurl.com/goofcurly2

**Reply by alcradite** [reply]
Like I said, the second link doesn't work. Here's what it says
"joflmkccibkooplaeoinecjbmdebglab is blocked
 This page has been blocked by Chrome 
 ERR_BLOCKED_BY_CLIENT"
 
![image](https://github.com/3kh0/ext-remover/assets/133121631/c3321bb3-8112-44b6-9848-479da7d023ad)
And when you refresh or press reload, it will bring you to "chrome-extension://invalid/" which will say
"invalid is blocked
This page has been blocked by Chrome
ERR_BLOCKED_BY_CLIENT"
![image](https://github.com/3kh0/ext-remover/assets/133121631/e70cd32a-7de4-4ec9-9876-a5d6daeb24c2)
But thanks for the suggestion!

**Reply by ghost** [reply]
that page will show if the extension isn't installed. are you sure that you have the right blocker?

**Comment by 2030mkitchin** [comment]
same thing happend to me

**Reply by funkymokey** [reply]
your a little late 😭

**Reply by S-PScripts** [reply]
he's just a lil lost ok
